app/v1/models/des/DesBroughtForwardLoss.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.des
+
+import play.api.libs.json._
+import v1.models.domain.BroughtForwardLoss
+import v1.models.requestData.DesTaxYear
+
+case class DesBroughtForwardLoss(lossType: String,
+                                 incomeSourceId: Option[String],
+                                 taxYearBroughtForwardFrom: String,
+                                 broughtForwardLossAmount: BigDecimal){
+
+  def toDes(desBroughtForwardLoss: DesBroughtForwardLoss) : BroughtForwardLoss = {
```

Comment:
Should this be `fromDes` or something similar? I think it would make more sense, since `toDes` to me suggests `MTD -> DES` rather than the other way around.

Reply:
:man_facepalming: